spacetime/tests/test_norm.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
```

Comment:
I think the file should use a clearer name indicating what problem it is trying to solve / test.  Module-level docstrings will be good too.